tests/tests/cpld_bmark_p7_35d

```diff
@@ -17,10 +17,10 @@ export SHOUR=00
 export SECS=`expr $SHOUR \* 3600`
 export BMIC=.true.
 
-export DAYS=35
```

Comment:
Did you mean to change to DAYS=1 and FHMAX=24? If commented out, what values are used?

Reply:
DAYS=1